packages/scalecube-microservice/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@scalecube/scalecube-microservice",
+  "version": "1.0.0",
+  "private": false,
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged"
+    }
+  },
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "lint": "tslint -c ../../tslint.json './src/**/*.(ts|tsx|js)'"
```

Comment:
try to use just "tslint -p tsconfig.json".
It should find tslint.json in the root automatically and will figure out what files to validate from tsconfig.json